SparkPost/2web2ui
# TR-1285: Live previewing for prototype fullscreen code editor

Refer to subtask [TR-1285](https://jira.int.messagesystems.com/browse/TR-1285).

### What Changed

- The preview automatically updates when content on left is modified.

![amp_preview](https://user-images.githubusercontent.com/1335605/57956514-38e70500-78c7-11e9-9fa5-62e3006fbabd.gif)

- Display an error with some guidance if there is a preview error.

<img width="1391" alt="Screen Shot 2019-05-20 at 10 07 58 AM" src="https://user-images.githubusercontent.com/1335605/58027673-2f3be800-7ae7-11e9-8c41-7c9159678a78.png">

*** I was going to surface line errors in the editors margin as annotations, but found that our API only returns the first error.

### How To Test

1. Modify some content in next editor — should update content in preview
1. Update content with `{{ break; }}` — should present full panel error message
